README.md

```diff
@@ -1,28 +1 @@
-# Promise Katas
-This material accompanies the **Working with asynchronous Javascript** lecture.
-
-## How to set up
-
-Clone and run `npm install`
-
-To run the tests run `npm t` or `npm test`
-
-## Instructions
-
-Do not change the test files. 
-
-Do not change the code in fakeApi.js.
-
-Do not change the name of the functions as these are imported into the test files. 
-
-Read the instructions in the comments above each function in both promise-katas-1.js and promise-katas-2.js.
-
-Edit the function body in both promise-katas-1.js and promise-katas-2.js to pass all of the tests.
-
-You can read the fakeApi to understand what data will get returned to you. The fakeApi is set up as if it was a real API. 
-You should always check what data is returned to you and the level of nesting in order for you to access the information 
-you need from the data that is returned. 
-
-You can read the test files to understand what is being tested and how to pass them. 
-
-If you do not understand an exercise, or get stuck, ask a mentor for help. 
+Promise Katas completed based on the WOrking with asynchronous Javascript lecture.
```

Comment:
Should always include some basic info about installation, what the repo is for, how to install it and how to use it at the bare minimum